src/GCodes/GCodeException.h

```diff
@@ -50,6 +63,7 @@ class GCodeException
 	int line;
 	int column;
 	const char *_ecv_array _ecv_null message;
+	GCodeExceptionSource source;
```

Comment:
Would it be worth storing commandLetter and commandNumber here too? We could use a dummy letter (e.g. 0) if there isn't one available. Then we wouldn't need to pass a GCodeBuffer to GetMessage.

Reply:
Possibly. I'm not sure if any of the `GCodeException`s that are initialised with a string only use a `gb` in their respective `GetMessage` call. Removing `gb` from the `GetMessage` method might in that case remove the command from the error message.